execution-engine-commons/src/main/java/com/odysseusinc/arachne/execution_engine_common/api/v1/dto/DataSourceUnsecuredDTO.java

```diff
@@ -200,14 +200,14 @@ public void setKrbUser(String krbUser) {
         this.krbUser = krbUser;
     }
 
-    public byte[] getKrbKeytab() {
+    public byte[] getKeyfile() {
```

Comment:
Will break current WebAPI relaying on 1.14-SNAPSHOT. Can we leave old methods as deprecated?